RubberduckTests/Inspections/ImplicitActiveSheetReferenceInspectionTests.cs

```diff
@@ -106,21 +125,33 @@ public void ImplicitActiveSheetReference_ReportsInWorkbookModules()
     arr1 = Cells(1,2)
 End Sub
 ";
-            var module = ("Sheet1", inputCode, ComponentType.Document);
-            var vbe = MockVbeBuilder.BuildFromModules(module, ReferenceLibrary.Excel).Object;
-
-            using (var state = MockParser.CreateAndParse(vbe))
+            var modules = new (string, string, ComponentType)[]
             {
-                var documentModule = state.DeclarationFinder.UserDeclarations(DeclarationType.Document)
-                    .OfType<DocumentModuleDeclaration>()
-                    .Single();
-                documentModule.AddSupertypeName("Workbook");
-
-                var inspection = InspectionUnderTest(state);
-                var inspectionResults = inspection.GetInspectionResults(CancellationToken.None);
+                ("ThisWorkbook", inputCode, ComponentType.Document),
+                ("Sheet1", string.Empty, ComponentType.Document),
+                ("Class1", string.Empty, ComponentType.ClassModule)
+            };
+            Assert.AreEqual(1, InspectionResultsForModules(modules, ReferenceLibrary.Excel, DefaultDocumentModuleSupertypeNames).Count());
+        }
 
-                Assert.AreEqual(1, inspectionResults.Count());
-            }
+        [Test]
+        [Category("Inspections")]
+        public void ImplicitActiveSheetReference_NoResultForWorksheetVariable()
+        {
+            const string inputCode =
+                @"Sub foo()
+    Dim sh As Worksheet
+    Set sh = Sheet1
+    Debug.Print sh.Cells(1, 1)
+End Sub
+";
+            var modules = new (string, string, ComponentType)[]
+            {
+                ("ThisWorkbook", string.Empty, ComponentType.Document),
```

Comment:
Here, `ThisWorkbook` is not relevant to the test.

Reply:
It's not, but any real-life VBA project hosted in Excel is always going to have a `ThisWorkbook` and at least one `Worksheet` module. Not including them would make the tests work in a hypothetical scenario that cannot happen (e.g. an Excel-hosted VBA project without a `ThisWorkbook` module).